File: examples/anthropic-claude-code-in-sandbox-python/anthropic_claude_code_in_sandbox/build_dev.py
from dotenv import load_dotenv
from e2b import Template, default_build_logger
from .template import template, template_name

load_dotenv()

Template.build(
    template,
    alias=f"{template_name}-dev",
    cpu_count=1,
    memory_mb=1024,
    on_build_logs=default_build_logger(),
)

File: examples/anthropic-claude-code-in-sandbox-js/src/build.dev.ts
import "dotenv/config";
import { Template, defaultBuildLogger } from "e2b";
import { template, templateName } from "./template";

async function main() {
  await Template.build(template, {
    alias: `${templateName}-dev`,
    cpuCount: 1,
    memoryMB: 1024,
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(console.error);
